ITE/4/README.adoc

```diff
@@ -197,11 +194,13 @@ their own, making them part of the supply chain, or metadata is replaced by
 authorized functionaries. The security implications of frequently changing
 generic resources are discussed in the Security section below.
 
-==== General Structure for URI Resolvers
+==== URI Resolvers
 
-While the implementations of resolvers will vary wildly depending on the context
-they're written for, this ITE recommends they follow a certain structure that
-enables them to be reused by different stakeholders looking to hash the same
+ITE-4 compliant in-toto tools will require a strategy for determining what resources
```

Comment:
I'm not very sure if "strategy" is the right word here. I think resolvers are more tangible than the word strategy implies, at least to me. Or perhaps I'm unfamiliar with this particular usage...

Reply:
I agree strategy is probably not the right word, but I'm unsure what is the right word because its unclear to me what they are. I actually struggled in editing this section, because I don't know if the resolvers are a tool, some added lines of code, some external to the system. If you can give me a better idea of what they are, I can come up with a clearer word to describe them.

Reply:
Currently, we envision them being separate components that can be plugged in to in-toto tools. I think just "component" in place of "strategy" handles it for now. I'll make a suggestion.

Reply:
```suggestion
ITE-4 compliant in-toto tools will require components for determining what resources
```

Reply:
I agree. Components is better. I made the suggestion, so accept if you like.

Reply:
I've made suggestions below and this one (and the next line) are at https://github.com/in-toto/ITE/pull/8#discussion_r414883465 and https://github.com/in-toto/ITE/pull/8#discussion_r414883564.